app/scripts/modules/google/serverGroup/configure/wizard/advancedSettings/advancedSettings.directive.html

```diff
@@ -34,6 +34,22 @@
 </div>
 <div class="form-group">
   <div class="sm-label-left">
+    <b>User Data (optional)</b>
+    <help-field key="gce.serverGroup.userData"></help-field>
+  </div>
+</div>
+<div class="form-group">
+  <div class="col-md-3 sm-label-right">User Data Value</div>
+  <div class="col-md-9 no-spel">
```

Comment:
I'd drop the `no-spel` attribute here and below - someone might want to use pipeline expressions here.